src/main/webapp/app/instructor-dashboard/instructor-dashboard.component.js

```diff
@@ -40,10 +39,6 @@
             return $filter('amDifference')(completionDate, initializationDate, 'minutes');
         }
 
```

Comment:
Why did you delete this? It is not related to the functionality in this pull request

Reply:
It is just a debugging function that I apparently forgot last time. All it does is  console.log("cleanup called");